server/test/controllers/userController.test.js

```diff
@@ -68,7 +68,7 @@ describe('UserController', () => {
     it('should return "please sign in" when trying to post message', (done) => {
       request(app).post('/api/v1/group/1/message')
         .end((err, res) => {
-          res.status.should.equal(400);
+          res.status.should.equal(401);
           res.body.should.have.property('message', res.body.message);
```

Comment:
it description not properly constructed. for example "it should return please sign in when trying to post message when not sign in"